src/app/zap-templates/zcl/data-model/chip/laundry-washer-cluster.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="SemanticTags" type="ENUM16">
+    <cluster code="0x0051"/>
+    <item value="0x4000" name="Normal"/>
+    <item value="0x4001" name="Delicate"/>
+    <item value="0x4002" name="Heavy"/>
+    <item value="0x4003" name="Whites"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Laundry Washer</name>
+    <code>0x0051</code>
+    <define>LAUNDRY_WASHER_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster is an alias of the Mode Select cluster, defining additional semantics and
+      namespaced enumerated values for laundry washer devices.</description>
+    <!-- Base data types -->
+    <attribute side="server" code="0x0000" define="MODE_DESCRIPTION" type="CHAR_STRING"                                     writable="false" optional="false" isNullable="false" length="64">Description</attribute>
```

Comment:
Does this also need `<globalAttribute side="either" code="0xFFFD" value="2"/>`?  Do the aliases just share the ClusterRevision of the thing being aliased?  I think they should.

Reply:
Updated in [0d6b3d89be2beba6d443811cafdce3d8e88a11c1](https://github.com/project-chip/connectedhomeip/pull/26508/commits/0d6b3d89be2beba6d443811cafdce3d8e88a11c1)